dist/api_data.py

```diff
@@ -774,6 +774,10 @@ def __cmp__(self, other):
         RPC server for primary processes and use RPC for secondary
         processes). <b>Not yet supported in WiredTiger</b>''',
         type='boolean'),
+    Config('optrack', 'true', r'''
```

Comment:
I'm inclined to set the default value to 'false' for this option since it generates so much information.

I also think we should move this configuration option into the `connection_runtime_config` section - so that the tracing can be enabled/disabled via a call to `WT_CONNECTION:reconfigure`. Maybe we should split that into a different ticket though?